### DISCUSSION TITLE
Last version Chrome broken: Cannot read proprietys of null

### DISCUSSION THREAD

**Discussion by pomporov** [dani-garcia/vaultwarden#4348] [Q&A]
Hello

since this morning (i think after chrome auto-upgrade) . 

<img width="482" alt="Capture d’écran 2024-02-14 à 10 06 48" src="https://github.com/dani-garcia/vaultwarden/assets/19282056/a272abe4-32f8-4bb3-9955-5c8c8eb6d784">

Last version on the server , last chrome and last extension version . 

(direct conntection via web works fine)

thanks

**Comment by pomporov** [comment]
I just tested exact same error via OSX native app .

**Comment by BlackDex** [comment]
Works fine for me.
Try to fully logout and login again (not lock).
There were some issues with the clients in some cases, which are solved when you fully logout.

**Comment by philhawthorne** [comment]
Just uninstalled the Brave/Chrome extension and re-installed. Same error

**Comment by BlackDex** [comment]
Which version of Vaultwarden and which version of the extension do you have?
I have the latest Vaultwarden v1.30.3-bb2412d0 (or `testing` actually) and v2024.2.0 of the extension.

**Reply by BlackDex** [reply]
btw, I installed Brave Version 1.62.162 Chromium: 121.0.6167.164, installed the extension and it works just fine.

**Reply by philhawthorne** [reply]
Confirming upgrading Vaultwarden fixed this for me. Must have been on an older version
